es-AR/firefox/new.lang

```diff
@@ -207,6 +207,6 @@ Dale a Firefox una puesta a punto
 # You should keep the translation consistent with Firefox:
 # http://transvision.mozfr.org/string/?entity=browser/chrome/browser/safeMode.dtd:refreshProfile.label&repo=beta
 ;Refresh Firefox
-Refrescar Firefox
+Actualizá Firefox
```

Comment:
Note that, at this point, the page is inconsistent with the UI actually shipping in Firefox for es-AR